table/block_based/block.cc

```diff
@@ -1062,8 +1062,8 @@ Block::Block(BlockContents&& contents, size_t read_amp_bytes_per_bit,
 
         uint16_t map_offset;
         data_block_hash_index_.Initialize(
-            contents.data.data(),
-            static_cast<uint16_t>(contents.data.size() -
+            contents_.data.data(),
```

Comment:
We could use the members `data_` and `size_` here

Reply:
That's a good catch, thank you for the suggestion!